From: Samuel Holland <samuel@sholland.org>
Date: Tue, 7 Apr 2020 23:11:43 -0500
Subject: [PATCH 098/194] Revert "drm/sun4i: tcon: Prepare and enable TCON
 channel 0 clock at init"

This reverts commit b14e945bda8ae227d1bf2b1837c0c4a61721cd1a.
---
 drivers/gpu/drm/sun4i/sun4i_tcon.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/sun4i/sun4i_tcon.c b/drivers/gpu/drm/sun4i/sun4i_tcon.c
index 9f06dec..409b5c11 100644
--- a/drivers/gpu/drm/sun4i/sun4i_tcon.c
+++ b/drivers/gpu/drm/sun4i/sun4i_tcon.c
@@ -791,7 +791,6 @@ static int sun4i_tcon_init_clocks(struct device *dev,
 			return PTR_ERR(tcon->sclk0);
 		}
 	}
-	clk_prepare_enable(tcon->sclk0);
 
 	if (tcon->quirks->has_channel_1) {
 		tcon->sclk1 = devm_clk_get(dev, "tcon-ch1");
@@ -806,7 +805,6 @@ static int sun4i_tcon_init_clocks(struct device *dev,
 
 static void sun4i_tcon_free_clocks(struct sun4i_tcon *tcon)
 {
-	clk_disable_unprepare(tcon->sclk0);
 	clk_disable_unprepare(tcon->clk);
 }
 
